src/scales/scale.linearbase.js

```diff
@@ -47,10 +47,10 @@ module.exports = function(Chart) {
 				}
 			}
 
-			if (me.min === me.max) {
```

Comment:
The previous commit is not enough. Say when we set `min: 10` (whatever larger than default max), will result in me.min > me.max here but not processed correctly.
I changed this to be based on the opts' min setting if exists.